src/entry.rs

```diff
@@ -82,18 +94,25 @@ pub struct EntryMetadata {
     pub lang: Option<String>,
     pub categories: Vec<String>,
     pub draft: bool,
+    pub values: YamlMapping,
 }
 
 impl EntryMetadata {
     pub fn read(path: &Path) -> eyre::Result<Self> {
-        let raw = RawEntryMetadata::read(path).wrap_err(format!(
+        let raw_value = RawEntryMetadata::read_as_value(path).wrap_err(format!(
```

Comment:
This was done to separate out the standard fields from custom ones. Now that I put everything under a values key, this should be reverted back to how it was.